packages/client-sdk-nodejs/src/config/middleware/experimental-metrics-logging-middleware.ts

```diff
@@ -50,5 +112,42 @@ export class ExperimentalMetricsLoggingMiddleware extends ExperimentalMetricsMid
       (p, l, c) =>
         new ExperimentalMetricsLoggingMiddlewareRequestHandler(p, l, c)
     );
+    if (!ExperimentalMetricsLoggingMiddleware.isLoggingStarted) {
+      ExperimentalMetricsLoggingMiddleware.startLogging(this.logger);
+    }
+  }
+
+  private static startLogging(logger: MomentoLogger): void {
+    this.eldMonitor = monitorEventLoopDelay({
+      resolution: this.eventLoopDelayInterval,
+    });
+    this.eldMonitor.enable();
+    this.elu = performance.eventLoopUtilization();
+
+    setTimeout(() => {
```

Comment:
why do you need this outer `setTimeout`?

Reply:
I wanted the first output to occur one second in since there won't be any useful data right away.

Reply:
doesn't setInterval already do that

Reply:
```
[Parameters](https://developer.mozilla.org/en-US/docs/Web/API/setInterval#parameters)

func

    A [function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function) to be executed every delay milliseconds. The first execution happens after delay milliseconds.
```

https://developer.mozilla.org/en-US/docs/Web/API/setInterval

Reply:
🤦